schemas/undernames.js

```diff
@@ -0,0 +1,24 @@
+const increaseUndernameCountSchema = {
+  $id: '#/definitions/increaseUndernameCount',
+  type: 'object',
+  properties: {
+    function: {
+      type: 'string',
+      const: 'increaseUndernameCount',
+    },
+    name: {
+      type: 'string',
+      pattern: '^([a-zA-Z0-9][a-zA-Z0-9-]{0,49}[a-zA-Z0-9]|[a-zA-Z0-9]{1})$',
+    },
+    qty: {
+      type: 'number',
+      minimum: 1,
```

Comment:
nit: add maximum

Reply:
which would be `9990` with a total cap of 10k

Reply:
added, also added a comment both in the schema and on the undername constants to make sure to update the schema if the constants change.